Makefile

```diff
@@ -47,8 +47,10 @@ generate:
 	$(MAKE) -C api gen-files
 	$(MAKE) -C libcalico-go gen-files
 	$(MAKE) -C felix gen-files
+	$(MAKE) -C calicoctl gen-crds
 	$(MAKE) -C app-policy protobuf
 	$(MAKE) gen-manifests
+	$(MAKE) go-fmt
```

Comment:
Should this go inside the gen-crds target?  Otherwise, folks who run `make gen-crds` will get confused because CI runs `gen-crds` _and_ `fmt`